astroidsClient/www/ts/Asteroid.ts

```diff
@@ -111,19 +123,19 @@ module Astroids {
             }
 
             var percentageToApply: number = this.game.time.elapsed / this.timeToApplyCorrectionUpdate;
-            
+
             var deltaX = this.correctionUpdate.x * percentageToApply;
```

Comment:
3 times (almost) code duplication here... this isn't best practice but will be merged anyway.